src/modules/claims/claims.service.ts

```diff
@@ -104,6 +104,12 @@ export class ClaimsService {
         return this._cacheClient.getClaimsBySubject(did, { isAccepted, namespace });
     }
 
+    /**
+     * deleteClaim
+     *
+     * @description allows subject to request for credential
+     *
```

Comment:
```suggestion
     * @description allows subject to request for credential
```

Reply:
I followed already created convention in previous methods. Ill remove them.

Reply:
Ah sorry, I didn't realize that. Maybe there is a reason for them then 🤔 ? I think best to remove them for now though, thanks